components/brave_wallet/browser/zcash/rust/Cargo.toml

```diff
@@ -13,9 +13,7 @@ zcash_primitives  = { version = "0.15.1", default-features = false }
 zcash_note_encryption = "0.4"
 zcash_client_backend = { version = "0.12.1", default-features = false }
 shardtree = { version="0.3.2", features=["legacy-api"] }
-
-[dev-dependencies.paste]
-version = "1"
+paste = "1"
```

Comment:
why do we change it from dev-deps to deps?

Reply:
https://bravesoftware.slack.com/archives/C8ZJB9F9B/p1742564013653629?thread_ts=1742563801.954299&cid=C8ZJB9F9B